Tasks/AzureAppServiceSettingsV1/package.json

```diff
@@ -21,7 +21,9 @@
     "@types/node": "^16.11.39",
     "@types/q": "1.0.7",
     "@types/uuid": "^8.3.0",
-    "azure-pipelines-tasks-azurermdeploycommon": "^3.207.0",
+    "azure-devops-node-api": "11.2.0",
+    "azure-pipelines-task-lib": "^4.1.0",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.220.2",
```

Comment:
Latest version is `3.221.1`

Reply:
It is, but it contains Kudu fixes that are not needed in this tasks.

Reply:
There are 3 versions after `^3.220.2` ; they might contain some other fixes. Also, I recommend to keep versions consistent with our other PRs

Reply:
Done